resource-aggregate/events/resourceStateSnapshotTaken.go

```diff
@@ -712,6 +753,13 @@ func (e *ResourceStateSnapshotTaken) HandleCommand(ctx context.Context, cmd aggr
 		}
 		return []eventstore.Event{&rc}, nil
 	case *commands.ConfirmResourceDeleteRequest:
+		if req.GetCommandMetadata() == nil {
+			return nil, status.Errorf(codes.InvalidArgument, errInvalidCommandMetadata)
+		}
+		if req.GetStatus() == commands.Status_OK || req.GetStatus() == commands.Status_ACCEPTED {
+			// deleted can comes first than unpublish so we call same handler
```

Comment:
Deleted event might be received before ... better formulation please

Reply:
removed